__tests__/app.test.js

```diff
@@ -1,10 +1,15 @@
 const endpointsJson = require("../endpoints.json");
-/* Set up your test imports here */
+const request = require("supertest");
+const app = require("../app.js");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/");
+const db = require("../db/connection");
 
-/* Set up your beforeEach & afterAll functions here */
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
 
 describe("GET /api", () => {
-  test("200: Responds with an object detailing the documentation for each endpoint", () => {
+  test.only("200: Responds with an object detailing the documentation for each endpoint", () => {
```

Comment:
We don't want to push up when we have a .only on our tests, just fyi. It can lead to false positives - fine here, but just for your next few tasks :)